lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -72,6 +73,16 @@ function requestHandler(request, response) {
         delay = parseInt(queryString.delay, 10) || 2000;
       }
 
+      if (typeof queryString.extra_header !== 'undefined') {
+        let extraHeaders = queryString.extra_header;
+        extraHeaders = Array.isArray(extraHeaders) ? extraHeaders : [extraHeaders];
+
+        extraHeaders.forEach(header => {
+          const parts = header.split(':');
+          headers[parts[0]] = parts.slice(1).join(':');
```

Comment:
this might be complete overkill, but can we make a set of allowed headers and only add to `headers` if found in there? We block hidden files and anything outside of the working directory, but you never know...

Reply:
One can't be too careful!